lib/logstash/util/compress.rb

```diff
@@ -0,0 +1,109 @@
+# encoding: utf-8
+require "logstash/namespace"
+require "logstash/environment"
+require "zip"
+require "rubygems/package"
+require "fileutils"
+require "tempfile"
+require "zlib"
+
+module LogStash::Util
+  module Zip
+
+    extend self
+
+    # Extract a zip file into a destination directory.
+    # @param source [String] The location of the file to extract
+    # @param target [String] Where you do want the file to be extracted
+    def extract(source, target)
```

Comment:
can we also document what happen if any target file previously exist? it overwrites?